src/libraries/System.Runtime.Serialization.Xml/ref/System.Runtime.Serialization.Xml.cs

```diff
@@ -75,6 +119,41 @@ public partial class DataContractSerializerSettings
         public System.Xml.XmlDictionaryString? RootNamespace { get { throw null; } set { } }
         public bool SerializeReadOnlyTypes { get { throw null; } set { } }
     }
+    public sealed partial class DataContractSet
+    {
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Data Contract Serialization and Deserialization might require types that cannot be statically analyzed. Make sure all of the required types are preserved.")]
+        public void Add(Type type) { throw null; }
+        public static void CompileSchemaSet(System.Xml.Schema.XmlSchemaSet schemaSet) { throw null; }
+        public DataContractSet(ISerializationSurrogateProvider? dataContractSurrogate, System.Collections.Generic.ICollection<Type>? referencedTypes, System.Collections.Generic.ICollection<Type>? referencedCollectionTypes) { throw null; }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Data Contract Serialization and Deserialization might require types that cannot be statically analyzed. Make sure all of the required types are preserved.")]
+        public DataContractSet(DataContractSet dataContractSet) { throw null; }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Data Contract Serialization and Deserialization might require types that cannot be statically analyzed. Make sure all of the required types are preserved.")]
+        public void ExportSchemaSet(System.Xml.Schema.XmlSchemaSet schemaSet) { throw null; }
+        [System.Diagnostics.CodeAnalysis.RequiresUnreferencedCodeAttribute("Data Contract Serialization and Deserialization might require types that cannot be statically analyzed. Make sure all of the required types are preserved.")]
```

Comment:
I see this is the result of refactoring. Was this needed to avoid exposing SchemaExporter?

Reply:
Yeah. Compile/Export/ImportSchemaSet were added so we could keep SchemaExporter (and SchemaImporter brought back from 4.8) internal. I tried keeping as much as possible internal with DCS, only putting the public schema API's and CodeDom stuff in the new 'external' package.